extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -0,0 +1,91 @@
+/*global require*/
+/*jslint newcap:true*/
+/*jshint camelcase:false*/
+/*jshint maxlen:200*/
+require([
+	'ext.wikia.adEngine.adEngine',
+	'ext.wikia.adEngine.adLogicHighValueCountry',
+	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.config.desktop',
+	'ext.wikia.adEngine.customAdsLoader',
+	'ext.wikia.adEngine.dartHelper',
+	'ext.wikia.adEngine.messageListener',
+	'ext.wikia.adEngine.provider.evolve',
+	'ext.wikia.adEngine.slotTracker',
+	'ext.wikia.adEngine.slotTweaker',
+	'wikia.krux',
+	'wikia.window'
+], function (
+	adEngine,
+	adLogicHighValueCountry,
+	adTracker,
+	adConfigDesktop,
+	customAdsLoader,
+	dartHelper,
+	messageListener,
+	providerEvolve,
+	slotTracker,
+	slotTweaker,
+	krux,
+	window
+) {
+	'use strict';
+
+	var kruxSiteId = 'JU3_GW1b';
+
+	window.AdEngine_getTrackerStats = slotTracker.getStats;
+
+	// DART API for Liftium
+	window.LiftiumDART = {
+		getUrl: function (slotname, slotsize) {
+			if (slotsize) {
+				slotsize += ',1x1';
+			}
+			return dartHelper.getUrl({
+				slotname: slotname,
+				slotsize: slotsize,
+				adType: 'adi',
+				src: 'liftium'
+			});
+		}
+	};
+
+	messageListener.init();
+
+	// Register Evolve hop
+	window.evolve_hop = providerEvolve.hop;
+
+	// Register window.wikiaDartHelper so jwplayer can use it
+	window.wikiaDartHelper = dartHelper;
+
+	// Register adLogicHighValueCountry as so Liftium can use it
+	window.adLogicHighValueCountry = adLogicHighValueCountry;
+
+	// Register adSlotTweaker so DART creatives can use it
+	// https://www.google.com/dfp/5441#delivery/CreateCreativeTemplate/creativeTemplateId=10017012
+	window.adSlotTweaker = slotTweaker;
+
+	// Custom ads (skins, footer, etc)
+	window.loadCustomAd = customAdsLoader.loadCustomAd;
+
+	// Everything starts after content and JS
+	window.wgAfterContentAndJS.push(function () {
+		// Ads
+		adTracker.measureTime('adengine.init', 'queue.desktop').track();
+		window.adslots2 = window.adslots2 || [];
+		adEngine.run(adConfigDesktop, window.adslots2, 'queue.desktop');
+
+		// Krux
+		krux.load(kruxSiteId);
```

Comment:
I think I see why you removed `wikia.log`. If we still want to use it it'll be used only here, to inform us that the code is about to load `krux`. I think, I agree and we don't need `wikia.log` for this - we can see if `krux` was loaded by checking `localStorage`.